README.md

```diff
@@ -2,36 +2,47 @@
 
 # dc-extension-hierarchy-chooser
 
-> Allows selection of content links and content references from a hierarchy tree.
+> Allows users to browse and add content items from a hierarchy rather than having to find these items in the Content Library
+![Browse hierarcy](https://cdn.media.amplience.net/i/ampproduct/add-existing-node-4?w=1600&fmt=png)
```

Comment:
Thanks for the changes. Would be nice to have a screenshot of the extension itself as well.